networkdb/cluster.go

```diff
@@ -449,8 +453,9 @@ func (nDB *NetworkDB) bulkSyncNode(networks []string, node string, unsolicited b
 	// Wait on a response only if it is unsolicited.
 	if unsolicited {
 		startTime := time.Now()
+		t := time.NewTimer(30 * time.Second)
 		select {
-		case <-time.After(30 * time.Second):
```

Comment:
It looks like you are using `<-t.C` instead of `After`. But both seems identical from a functionality standpoint. Why is this change needed ?